src/drivers/apl-ssp.c

```diff
@@ -242,18 +243,26 @@ static inline int ssp_set_config(struct dai *dai,
 		inverted_frame = true; /* handled later with format */
 		break;
 	case SOF_DAI_FMT_IB_IF:
-		sspsp |= SSPSP_SCMODE(2);
+		inverted_clock = true; /* handled later with bclk idle */
 		inverted_frame = true; /* handled later with format */
 		break;
 	case SOF_DAI_FMT_IB_NF:
-		sspsp |= SSPSP_SCMODE(2);
+		inverted_clock = true; /* handled later with bclk idle */
 		break;
 	default:
 		trace_ssp_error("ec3");
 		ret = -EINVAL;
 		goto out;
 	}
 
+	/* supporting bclk idle state */
+	if (ssp->params.clks_control &
+		SOF_DAI_INTEL_SSP_CLKCTRL_BCLK_IDLE) {
```

Comment:
it's really BCLK_IDLE_LOW?

Reply:
I've added comments.

Reply:
might as well call the value SOF_DAI_INTEL_SSP_CLKCTRL_BCLK_IDLE_HIGH to make it self-explanatory.